nimble/host/src/ble_gap.c

```diff
@@ -2520,12 +2524,6 @@ ble_gap_ext_adv_configure(uint8_t instance,
 
     ble_hs_lock();
 
-    /* TODO should we allow to reconfigure existing instance? */
-    if (ble_gap_slave[instance].configured) {
```

Comment:
I don't think we should allow to reconfigure active instance, should this be checked here?

Reply:
I think we can allow reconfiguration as long as we didn't enable extended advertising.